docs/citing/index.rst

```diff
@@ -57,3 +71,17 @@ If you use OSQP for published work, we encourage you to put a star on `GitHub <h
             doi       = {10.1109/CDC.2017.8263928},
             url       = {https://doi.org/10.1109/CDC.2017.8263928},
           }
+
+    Mixed-integer optimization
+        A branch-and-bound solver for mixed-integer quadratic optimization in this `paper <https://stellato.io/assets/downloads/publications/2018/miosqp_ecc.pdf>`__.
```

Comment:
@bstellato Can you please fix this link? Also related to https://github.com/osqp/osqp/issues/329

Reply:
The new link should be `https://stellato.io/downloads/publications/2018/miosqp_ecc.pdf`.

Reply:
Done here - will send separate PR for master.

Reply:
Please update the link here as well:
https://github.com/osqp/osqp/blob/master/site/_pages/citing.md

Reply:
Fixed in this branch. Will fix in master separately.